src/main/java/net/neoforged/neoforge/common/NeoForgeMod.java

```diff
@@ -493,6 +496,13 @@ public static void enableMilkFluid() {
         enableMilkFluid = true;
     }
 
+    /**
+     * Run this during mod construction to enable merged attribute tooltip functionality.
+     */
+    public static void enableMergedAttributeTooltips() {
```

Comment:
Is there a reason to have this be a code hook rather than potentially a config option?

On a side note, given it seems we are getting a decent number of these enable styled flags, do we potentially want to in the next breaking cycle move them to their own class that just contains said flags, and also potentially mark in logs (or at least crash logs which ones are enabled)?

Reply:
The code hook allows mods to enable it on request, which a config cannot do.